server.js

```diff
@@ -0,0 +1,19 @@
+// Server setup
+const express = require('express')
+const app = express()
+const path = require('path')
+const api = require('./server/routes/api')
+let bodyParser = require('body-parser')
+
+// Mongoose setup
+const mongoose = require('mongoose')
+mongoose.connect('mongodb://localhost/mongoose-expense', { useNewUrlParser: true })
+
+app.use(express.static(path.join(__dirname, 'dist')))
+app.use(express.static(path.join(__dirname, 'node_modules')))
```

Comment:
to handle the `req.body` of POST request, you should add this line of code before the `app.use('/', api)`:
`app.use(express.json())`